cli/tools/registry/provenance.rs

```diff
@@ -427,9 +427,11 @@ struct SigningCertificateResponse {
 impl FulcioSigner {
   pub fn new() -> Result<Self, AnyError> {
     let rng = SystemRandom::new();
-    let document = EcdsaKeyPair::generate_pkcs8(ALGORITHM, &rng)?;
+    let document = EcdsaKeyPair::generate_pkcs8(ALGORITHM, &rng)
```

Comment:
Are these intended to be part of the PR?

Reply:
Yeah, this works fine when deno_crypto is not included as dependency.